src/components/multi-action-button/multi-action-button.test.js

```diff
@@ -181,290 +181,308 @@ context("Tests for MultiActionButton component", () => {
-          multiActionButtonList().eq(0).focus();
-          multiActionButtonListContainer()
-            .eq(0)
-            .trigger("keydown", { ctrlKey: true, key: "ArrowDown" });
-          multiActionButtonList().eq(2).should("be.focused");
-        });
-    });
+    multiActionButton()
+      .eq(0)
+      .trigger("mouseover")
+      .then(() => {
+        multiActionButtonList().eq(0).focus();
+        multiActionButtonListContainer()
+          .eq(0)
+          .trigger("keydown", { metaKey: true, key: "ArrowDown" });
+        multiActionButtonList().eq(2).should("be.focused");
+      });
   });
+});
 
-  describe("pressing End while MultiActionButton is open", () => {
-    it("should move focus to last child button", () => {
-      CypressMountWithProviders(
-        <>
-          <MultiActionButtonList />
-          <MultiActionButtonList />
-        </>
-      );
+describe("pressing ctrlKey + ArrowDown while MultiActionButton is open", () => {
+  it("should move focus to last child button", () => {
+    CypressMountWithProviders(
+      <>
+        <MultiActionButtonList />
+        <MultiActionButtonList />
+      </>
+    );
 
-      multiActionButton()
-        .eq(0)
-        .trigger("mouseover")
-        .then(() => {
-          multiActionButtonList().eq(0).focus();
-          multiActionButtonListContainer()
-            .eq(0)
-            .trigger("keydown", keyCode("End"));
-          multiActionButtonList().eq(2).should("be.focused");
-        });
-    });
+    multiActionButton()
+      .eq(0)
+      .trigger("mouseover")
+      .then(() => {
+        multiActionButtonList().eq(0).focus();
+        multiActionButtonListContainer()
+          .eq(0)
+          .trigger("keydown", { ctrlKey: true, key: "ArrowDown" });
+        multiActionButtonList().eq(2).should("be.focused");
+      });
   });
+});
 
-  describe("Pressing esc while MultiActionButton is open", () => {
-    it("should close MultiActionButton", () => {
-      CypressMountWithProviders(<MultiActionButtonList />);
+describe("pressing End while MultiActionButton is open", () => {
+  it("should move focus to last child button", () => {
+    CypressMountWithProviders(
+      <>
+        <MultiActionButtonList />
+        <MultiActionButtonList />
+      </>
+    );
 
-      multiActionButton()
-        .eq(0)
-        .trigger("mouseover")
-        .then(() => {
-          multiActionButtonList().eq(1).focus();
-          multiActionButton().eq(0).trigger("keydown", keyCode("Esc"));
-          multiActionButtonListContainer().should("not.exist");
-        });
-    });
+    multiActionButton()
+      .eq(0)
+      .trigger("mouseover")
+      .then(() => {
+        multiActionButtonList().eq(0).focus();
+        multiActionButtonListContainer()
+          .eq(0)
+          .trigger("keydown", keyCode("End"));
+        multiActionButtonList().eq(2).should("be.focused");
+      });
+  });
+});
+
+describe("Pressing esc while MultiActionButton is open", () => {
```

Comment:
describe("pressing esc while MultiActionButton is open", () => {, change P to p to keep the consistency of all tests please